src/archanaNJun20/Bankingsystem.java

```diff
@@ -0,0 +1,66 @@
+package archanaNJun20;
+
+public class Bankingsystem {
+
+	int balance=1000;
+	int debit;
+	int credit;
+	int pBalance;
+	static int sBalance=1000;
+	static int sDebit;
+	static int sCredit;
+	static int sPBalance;
+	
+	void debitAmount(int pDebit) {
+		
+		balance=balance-pDebit;
+		debit++;
+		sDebit++;
+	}
+	void creditAmount(int pCredit) {
+		balance=balance+pCredit;
+		credit++;
+		sCredit++;
+	}
+	void printBalance() {
+		pBalance++;
+		sPBalance++;
```

Comment:
Why are you not printing Balance in this method?

Reply:
done